Cargo.toml

```diff
@@ -55,3 +55,8 @@ strip = false
 debug = 1
 lto = "thin"
 codegen-units = 256
+
+# fuser contains memory safety issues which trigger leaks and UB inside libfuse.so
+# It's used only as dev dependency in the test code, so this is okay.
+[patch.crates-io]
+fuser = { git = "https://github.com/In-line/fuser.git", rev = "8daba15728da46d9299a2b23daa3aeb9954e114e" }
```

Comment:
Please don't point to your personal fork, i'd rather have the crate ignored by the sanitizer until the fix

Reply:
Your PR was merged, you can point to fuser master for now